portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -103,12 +112,23 @@ private Entity createEntity(HttpServletRequest request) {
       long timestamp = (long) entity.getProperty("timestamp");
       String name = (String) entity.getProperty("name");
       String email = (String) entity.getProperty("email");
-      String comment = (String) entity.getProperty("comment");
-      
+      String comment = translateComment(request, entity);
       Post post = new Post(id, name, email, comment, timestamp);
      
       posts.add(post);
     }  
     return posts;  
   }
+
+  private String translateComment(HttpServletRequest request, Entity entity) {
+    String originalComment = (String) entity.getProperty("comment");
```

Comment:
What happens if the property doesn't exist?

Reply:
Same solution as above.